### DISCUSSION TITLE
Question about EZINSAR's deformation tracking capability

### DISCUSSION THREAD

**Discussion by forgetsyl** [alexisInSAR/EZ-InSAR#93] [Help]
Hi, Alexis. 
1. Could you please inform me whether the EZINSAR results show deformation magnitude and rate in the line-of-sight direction? 
2. Is it possible to track deformation magnitude and rate in both vertical and horizontal directions? As you are aware, this is an interesting topic for investigating landslides, faults, and plate motions.
3. Can EZINSAR be used to investigate landslides?

best wishes.

**Comment by alexisInSAR** [comment]
Hi, 

1. Yes. 
2. No, you need to compute Asc. and Desc. tracks in order to inverse the vertical and horizontal components. Numerous examples are available in the literature. 
3. There are numerous examples for landslide monitoring using InSAR techniques, but all depends on your landslides (i.e., InSAR coherences, amplitudes of motions, motion directions, etc.). 

Kind regards, 

Alexis
